### PR TITLE
Adds https support, updates dashboard UI, installs passport.js, includes separate Dashboard route and page

### PR DESCRIPTION
**Pull request by ploper187**

### CODE REVIEW

File: RushMeAdmin/app/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/*
```
Comment (mmoftah) [inline RushMeAdmin/app/routes/index.js]: I could be wrong, but from what I understand about these changes you have made to the structure, this file would be accessible client-side. This seems like a lot of information is here that should not be, are we sure this is in the right place?

**Review by mmoftah — COMMENTED** [review]
I am skeptical only about the index.js route. Seems like that should all be in the server, but please let me know if I am misunderstanding it.

File: RushMeAdmin/app/routes/index.js (hunk 1 above)
Comment (ploper187) [reply RushMeAdmin/app/routes/index.js]: You're right; the information shouldn't be available client side. I threw depcrecated code in index.js, unsure where else to put it. I'll commit an empty index.js

**Comment by ploper187** [comment]
> I am skeptical only about the index.js route. Seems like that should all be in the server, but please let me know if I am misunderstanding it.

Are function definitions global? I'd love to make a server.js function for Cognito-authenticated DynamoDB sessions

File: RushMeAdmin/app/routes/index.js (hunk 1 above)
Comment (ploper187) [reply RushMeAdmin/app/routes/index.js]: See [474aa9e](https://github.com/RushMeTeam/RushMeBackend/pull/8/commits/474aa9ec53b605655d932f45befbbeee6068791d)

File: RushMeAdmin/app/routes/in/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// File: dashboard.controller.js
```
Comment (mmoftah) [inline RushMeAdmin/app/routes/in/dashboard.js]: Let's remember to remove this file now that we switched it all over.

**Comment by ploper187** [comment]
@mmoftah can you close the request? Looks like a solid start!

**Pull request merged**